www/modules/centreon-open-tickets/providers/GlpiRestApi/GlpiRestApiProvider.class.php

```diff
@@ -1387,7 +1387,9 @@ protected function createTicket($ticketArguments)
             'content' => $ticketArguments['content'],
             'entities_id' => $ticketArguments['entity'],
             'urgency' => $ticketArguments['urgency'],
-            'itilcategories_id' => $ticketArguments['category'],
+            'itilcategories_id' => (
+                    $ticketArguments['category'] != ''
+                ) ? $ticketArguments['category'] : 0,
```

Comment:
```suggestion
            'itilcategories_id' => ! empty($ticketArguments['category']) ? $ticketArguments['category'] : 0,
```

Reply:
empty() will check for empty string and null value it is better for handle those cases